example/src/main/config/qpid/qdrouterd-with-broker.json

```diff
@@ -118,6 +119,34 @@
       }
   }],
 
+  ["vhost", {
+    "id": "hono-mqtt-internal",
+    "maxConnections": 30,
+    "groups": {
+      "Hono": {
+        "users": "Eclipse IoT;Hono;mqtt-adapter",
+        "remoteHosts": "*",
+        "allowUserIdProxy": true,
+        "sources": "control/*",
+        "targets": "control/*"
+      }
+    }
+  }],
+
+  ["vhost", {
+    "id": "hono-http-internal",
+    "maxConnections": 30,
+    "groups": {
+      "Hono": {
+        "users": "Eclipse IoT;Hono;http-adapter",
```

Comment:
The defined name of the HTTP adapter is `hono-http` - maybe this should be used here as well?
Are these users additionally contained in a user database (e.g. `qdrouterd.sasldb`)?

Reply:
Ah, I see, thanks for the explanation :-)